docs/hashes.toml

```diff
@@ -134,7 +134,7 @@ index = 1
 
 # Spec: ckb_dev
 [ckb_dev]
-spec_hash = "0x360b413b576acc041b778da1a030cfc98fd5d934e2b12ab750aeee1e4ea21017"
+spec_hash = "0x4df07f7ae1bbc06373bcc3487ca8011ee3f57a3d46d2357b34728cbd412d4ce6"
```

Comment:
The original value in v0.43.0:

https://github.com/nervosnetwork/ckb/blob/bbbe5a58f7024f1157fba75f26ab4e2fdfee1e38/docs/hashes.toml#L135-L137

This value is changed because:
https://github.com/nervosnetwork/ckb/blob/eca92887aac2119ad677df013721e0336d8cb2b6/resource/specs/dev.toml#L98-L104